apps/meteor/client/views/admin/viewLogs/ViewLogsPage.tsx

```diff
@@ -1,18 +1,33 @@
+import type { IStats, Serialized } from '@rocket.chat/core-typings';
+import { Tabs } from '@rocket.chat/fuselage';
 import { useTranslation } from '@rocket.chat/ui-contexts';
 import type { ReactElement } from 'react';
-import React from 'react';
+import React, { useState } from 'react';
 
 import Page from '../../../components/Page';
+import AnalyticsReports from './AnalyticsReports';
 import ServerLogs from './ServerLogs';
+import { useAnalyticsObject } from './hooks/useAnalyticsObject';
 
 const ViewLogsPage = (): ReactElement => {
 	const t = useTranslation();
+	const { data, isSuccess } = useAnalyticsObject();
```

Comment:
You can fetch the data inside the `AnalyticsReport`

Reply:
If I did this it would re-run the query, and consequently the rest request, on every tab change, creating a lot of unnecessary re-requests. On this note I belive it would be good to refactor the ServerLogs component to avoid unnecessary requests as well, but since it is quite old and probably will be deprecated in the future I didn't try to.

Reply:
Why would be re-render? The useQuery has a cache mechanism that prevent that, it's just a good practice to use this hook, this was mentioned on the front-end meeting

Reply:
Also, we need to handle error scenario and refetch if necessary